README.md

```diff
@@ -68,6 +68,12 @@ Default: `100`
 
 Max count of items in a single chunk.
 
+### S3PIPELINE_PROCESS_ITEM_TIMEOUT (Optional)
+
+Default: `30.0`
```

Comment:
I think bigger default value, e.g. `300.0`, is preferable.

If a user uses settings `DOWNLOAD_DEALY = 3`, which is available in a comment  of Scrapy's [default project template](https://github.com/scrapy/scrapy/blob/2.4.0/scrapy/templates/project/module/settings.py.tmpl#L28), it will take 3 * 100 = 300 seconds to fill a chunk with 100 items.  It might be confusing if a number of items in an uploaded chunk is fewer than S3PIPELINE_MAX_CHUNK_SIZE by default.

Reply:
Well, in my case, I need to crawl more than 10 items per second each scrapy-redis node, so I set the deafult value to be `30.0`. But your suggestion is right, more people just use it to do a regular crawl work, it's better to be `300.0`.

Reply:
I'm sorry. I misunderstood the behavior of timer. Since timer is reset to zero when an item is processed, the previous default value `30.0` is big enough. So, please revert to `30.0`.